ShellScript/Bash.sublime-syntax

```diff
@@ -762,13 +762,23 @@ contexts:
   test-pattern-common:
     - match: (?=\s)
       pop: 1
-    - match: \(
-      scope:
-        meta.group.regexp.shell
-        punctuation.definition.group.begin.regexp.shell
+    - match: ([?*+@!])?(\()
+      captures:
+        1: keyword.operator.quantifier.regexp.shell
+        2: meta.group.regexp.shell
+           punctuation.definition.group.begin.regexp.shell
       push: expansions-pattern-group-body
+    - match: (\[)(?:([!^])|-)?(?=((?!\s\]\]).)*?\])
```

Comment:
Maybe consider `|- (?x:`ing this with comments? The parts all make sense, but I had to throw it in an editor to read it.